README.md

```diff
@@ -6,42 +6,48 @@
 
 This is a Chainlink plugin implementation for version `4.x` of [web3.js](https://github.com/web3/web3.js).
 
-###### Get it from the NPM Registry
+## Prerequisites
+
+-   :gear: [NodeJS](https://nodejs.org/) (LTS/Fermium)
+-   :toolbox: [Yarn](https://yarnpkg.com/) or [npm](https://www.npmjs.com/package/npm)
+
+## Installation
+
+You can install the package either using [npm](https://www.npmjs.com/package/web3) or using [Yarn](https://yarnpkg.com/package/web3)
+
+### Using npm
 
 ```bash
-yarn add web3-plugin-chainlink
+npm install web3
 ```
 
-## Prerequisites
+### Using Yarn
 
--   :gear: [NodeJS](https://nodejs.org/) (LTS/Fermium)
--   :toolbox: [Yarn](https://yarnpkg.com/)
+```bash
+yarn add web3
+```
 
-## Getting Started
+## Found an issue or have a question or suggestion
 
--   :writing_hand: If you have questions [submit an issue](https://github.com/web3/web3.js/issues/new) or join us on [Discord](https://discord.gg/yjyvFRP)
+-   :writing_hand: If you found an issue or have a question or suggestion [submit an issue](https://github.com/ChainSafe/web3.js-plugin-chainlink/issues/new) or join us on [Discord](https://discord.gg/yjyvFRP)
     ![Discord](https://img.shields.io/discord/593655374469660673.svg?label=Discord&logo=discord)
 
-After cloning the repo,
+## Run the tests
+
+You may like to run the tests and examin their code to see how to use the plugin. And this would be also useful if you are a plugin writer.
```

Comment:
```suggestion
You may like to run the tests and examine their code to see how to use the plugin. And this would be also useful if you are a plugin writer.
```